Components/avatar.css

```diff
@@ -3,36 +3,30 @@
     display: flex;
     flex-direction: row;
     padding: 1rem;
-    border: 1px solid var(--color-black);
+    border: 1px solid var(--color-gray-200);
     width: 60%;
 }
-
 .avatar{
     padding: 1rem;
     align-self: center;
 }
-
 .avatar-img{
     border-radius: 50%;
     height:100%;
     width:100%;
```

Comment:
add height: auto and have object-fit: cover here

Reply:
Object-fit: cover -- done,
height: auto, is affecting the avatar with badges one, it used to get out, hence didn't change.